docs/EN/index.rst

```diff
@@ -3,29 +3,34 @@ Welcome to the Android APS documentation
+AndroidAPS is a app that acts as an artificial pancreas system (APS) on an Android smartphone. What is an artificial pancreas system? It is a software program that aims to do what a living pancreas does: keep blood sugar levels within healthy limits automatically. An APS can't do the job as well as a biological pancreas does, but it can make type 1 diabetes easier to manage using devices that are commercially available and software that is simple and safe. Those devices include a continuous glucose monitor (CGM) to tell AndroidAPS about your blood sugar levels and an insulin pump which AndroidAPS controls to deliver appropriate doses of insulin. The app communicates with those devices via bluetooth. It makes its dosing calculations using an algorithm, or set of rules, developed for another artificial pancreas system, called OpenAPS, which has thousands of users and has accumulated millions of hours of use. 
+
+A note of caution: AndroidAPS is not regulated by any medical authority in any country. Using AndroidAPS is essentially carrying out a medical experiment on yourself. Setting up the system requires determination and technical knowledge. If you don't have the technical know-how at the beginning, you will by the end. All the information you need can be found in these documents, elsewhere online, or from others who have already done it -- you can ask them in Facebook groups or other forums. Many people have successfully built AndroidAPS and are now using it entirely safely, but it is essential that every user:
+* Builds the system themselves so that they thoroughly understand how it works
+* Adjusts the settings to suit their own diabetes
+* Maintains and monitors the system to ensure it is working properly
+If you're ready for the challenge, please read on. 
 
 **Primary goals behind AndroidAPS:**
 
-* modular app where it is possible and easy to add new modules without touching the rest of the code
-* app that allows localization
-* app where it is easy to select what will be included in final apk with an easy change and compilation
-* app which supports open and closed APS mode
-* app where you can see how APS works: input params, result and final decision
-* ability to add more APS algorithms and let the user decide what to use
-* app which is independent from a pump driver and contains a "Virtual pump" to allow users to safely play with APS
-* app with tight Nightscout integration
-* app where is easy to add/remove constraints for user safety
-* all-in-one app for managing T1D with APS and Nightscout
+* An app with safety built in. To read about the safety features of the algorithms, known as oref0 and oref1, click here (https://openaps.org/reference-design/)
+* An all-in-one app for managing type 1 diabetes with an artificial pancreas and Nightscout
+* An app to which users can easily add or remove modules as needed
+* An app that allows localization {I don't know what this means. Do you mean it can be used offline?}
```

Comment:
Hi, there some information is put in.
Localization is the step after internationalisation ;) (A bit more than just translations)

Reply:
Could we say something like: Available in multiple languages. Or is that simplifying too much?

Reply:
How about this: 
An app with different versions for specific locations and languages.